### PR TITLE
refactor: only formatting code on npm run format instead of npm run lint

### PR DESCRIPTION
**Pull request by ashin-czi**
- #3519

### Reviewers
**Functional:** @seve @nayib-jose-gloria @tihuan 

**Readability:** @seve @nayib-jose-gloria @tihuan 

---

## Changes
- add
- remove
    - `--fix` from `npm run lint` and moved to `npm run format` command
- modify

### CODE REVIEW

**Review by seve — APPROVED** [review]

File: frontend/package.json [hunk 1]
```diff
@@ -95,7 +95,7 @@
     "build": "NODE_OPTIONS=\"--max_old_space_size=2048\" next build",
     "start": "next start",
     "develop": "next dev",
-    "format": "npx prettier --write .",
+    "format": "concurrently \"npx prettier --write .\" \"node_modules/.bin/next lint --fix\" \"node_modules/.bin/stylelint --fix '**/*.{js,ts,tsx,css}'\"",
```
Comment (tihuan) [inline frontend/package.json:98]: Ooh thanks for adding `concurrently` here! I think this is a great speedup with probably a small chance of different commands competing to write to the same file? Don't think it'll happen any time soon, but maybe good for us to keep in mind 😆 !

File: frontend/src/components/common/Filter/common/style.ts [hunk 2]
```diff
@@ -35,7 +35,6 @@ export const SelectionIcon = styled(Icon)`
       width: 14px;
     }
   }
-}
```
Comment (tihuan) [inline frontend/src/components/common/Filter/common/style.ts:38]: woah we've had this invalid closing bracket for a while lol 😆 Thanks for fixing this!

**Review by tihuan — APPROVED** [review]
LGTM! Thanks so much for fixing this, @ashin-czi 🙏 ✨ 🙏 !

File: frontend/package.json (hunk 1 above)
Comment (ashin-czi) [reply frontend/package.json:98]: hmmm should we run the commands sequentially then? I think the small speed up might not be worth any collisions 😅

**Pull request merged**